examples/dataprocserverless-running-notebooks/README.md

```diff
@@ -0,0 +1,100 @@
+
+## File Directory Structure
+
+**dataproc_serverless**
+
+    ├── composer_input                   
+    │   ├── jobs/                       wrapper_papermill.py
+    │   ├── DAGs/                       serverless_airflow.py
+    ├── notebooks 
+    │   ├── datasets/                   electric_vehicle_population.csv
+    │   ├── jupyter/                    spark_notebook.ipynb
+    │   ├── jupyter/output              spark_notebook_outbook.ipynb
+    
+## File Details    
+### composer_input
+* **wrapper_papermill.py**: runs a papermill execution of input notebook and writes the output file into the assgined location
+* **serverless_airflow.py**: orchestrates the workflow 
+  * Dataproc Batch Creation :This operator runs your workloads on Dataproc Serverless
+    ```
+    create_batch1 = DataprocCreateBatchOperator()
+    create_batch2 = DataprocCreateBatchOperator()...
+    ```
+
+### notebooks 
+* **spark_notebook.ibynp** : This file creates Spark Session for Electric Vehicle Population, loads csv file in GCS and perform fundamental Spark functions
+    ```
+    spark = SparkSession \
+        .builder \
+        .appName("Spark Session for Electric Vehicle Population") \
+        .getOrCreate()
+    
+    path=f"gs://{gcs_bucket}/dataproc_serverless/notebooks/datasets/electric_vehicle_population.csv"
+    df = spark.read.csv(path, header=True)
+    df.show()
+    
+    df.filter(df['Model Year'] > 2020).show(5)
+    sortedByElectricRange = df.orderBy('Electric Range').show(10)
+    ```
+* **jupyter/output** : this is where papermill notebook execution outputs will be stored
+        Sample Notebook Output
+        ![Screenshot 2023-03-16 at 2 20 35 AM](https://user-images.githubusercontent.com/123537947/225531950-dccde663-fade-413d-9600-eb59ab669800.png)
+
+* **electric_vehicle_population.csv** : this truncated [dataset](https://catalog.data.gov/dataset/electric-vehicle-population-data) shows the Battery Electric Vehicles (BEVs) and Plug-in Hybrid Electric Vehicles (PHEVs) that are currently registered through Washington State Department of Licensing (DOL).
+
+# I) Orchestrating End to End Notebook Execution workflow in Cloud Composer 
+
+1. Make sure to modify gcs path for datasets in Notebook 
+
+2. Create [Persistent History Server](https://cloud.google.com/dataproc/docs/concepts/jobs/history-server)
+
+Make sure to follow the [guide](https://cloud.google.com/dataproc/docs/concepts/jobs/history-server#create_a_job_cluster) for setting up properties. For example, it must have the property 'spark:spark.history.fs.logDirectory' configured to act as a Spark history server
+
+3. Create a Cloud Composer Environment
+
+3. Find DAGs folder from Composer Environment and add serverless_airflow.py (DAGs file) to it in order to trigger DAGs execution:
+DAG folder from Cloud Composer Console 
+      
+4. Have all the files available in GCS bucket, except DAGs file which should go into your Composer DAGs folder
```

Comment:
- Manually, or via Continuous Integration, copy the notebook source code files to the appropriate GCS bucket

- Manually, or via Continuous Integration, copy the Airflow DAG files to your DAGs folder of your Cloud Composer environment GCS bucket